### ISSUE TITLE
I can't use facebook login and this package together

### ISSUE THREAD

**Issue by osmaneser** [oddbit/flutter_facebook_app_events#135]
Hi, firstly thank you this packet. 

I have a problem.

I can't pod install When i added facebook_app_events: ^0.11.1  in pubspec.yaml 

I get following errors

**Analyzing dependencies**
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    

    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 9.0.0)
      
    flutter_facebook_login (from `.symlinks/plugins/flutter_facebook_login/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 5.5)

**Comment by DennisAlund** [comment]
Please use the issue template and fill in the part of your expectations. 

You're using an old version of this plugin. Is that intentional in order to have same version of Facebook SDK as with the login plugin?

Try searching the issues here and stack overflow, I think that there was other issues with this that were resolved by using another login plugin.

**Comment by long1eu** [comment]
The issue is this there. Looks like you need to bump the dependency version.

```
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 11.1.0)

    flutter_facebook_auth (from `.symlinks/plugins/flutter_facebook_auth/ios`) was resolved to 3.5.1, which depends on
      FBSDKCoreKit (~> 11.2.0)
```
